<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskFlow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-10</y>
      <width>2051</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-20</y>
      <width>20</width>
      <height>1980</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Головна&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>  Головна</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="calendarButton">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>     Календар</string>
    </property>
   </widget>
   <widget class="QPushButton" name="foldersButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>190</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>       Статистика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="basketButton">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>Кошик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreatButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: none;
font: 16pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);
}
QPushButton:hover {
	color: rgb(255, 85, 0);
}</string>
    </property>
    <property name="text">
     <string>Створити        </string>
    </property>
   </widget>
   <widget class="QLabel" name="creatImg">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>15</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(223, 223, 223);
border: none;
font: 15pt &quot;Britannic Bold&quot;;
color:rgb(113, 114, 116);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgH">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgSt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>135</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ImgSearch">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>-40</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgS">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>556</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>   Налашт.</string>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgS_a">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgB_a">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>135</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgC_a">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgSt_a">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>580</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(197, 199, 203)</string>
    </property>
    <property name="text">
     <string>Версія: 0.21.12a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>170</y>
      <width>188</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>Інфо</string>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgI">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>165</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgI_a">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>165</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="foldButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>110</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
font: 18pt &quot;Britannic Bold&quot;;
color: rgb(56, 57, 58);</string>
    </property>
    <property name="text">
     <string>Папки </string>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonImgF_a">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>buttonImgI</zorder>
   <zorder>buttonImgI_a</zorder>
   <zorder>buttonImgC</zorder>
   <zorder>buttonImgB</zorder>
   <zorder>buttonImgSt</zorder>
   <zorder>ImgSearch</zorder>
   <zorder>buttonImgH</zorder>
   <zorder>creatImg</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>homeButton</zorder>
   <zorder>calendarButton</zorder>
   <zorder>foldersButton</zorder>
   <zorder>basketButton</zorder>
   <zorder>CreatButton</zorder>
   <zorder>searchLine</zorder>
   <zorder>buttonImgS</zorder>
   <zorder>settingsButton</zorder>
   <zorder>buttonImgS_a</zorder>
   <zorder>buttonImgB_a</zorder>
   <zorder>buttonImgC_a</zorder>
   <zorder>buttonImgSt_a</zorder>
   <zorder>label</zorder>
   <zorder>infoButton</zorder>
   <zorder>buttonImgF</zorder>
   <zorder>buttonImgF_a</zorder>
   <zorder>foldButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
